middle_end/flambda/simplify/simplify_expr.rec.ml

```diff
@@ -55,6 +55,22 @@ let get_and_place_lifted_constants dacc uacc scoping_rule
   in
   body, UA.with_r uacc r
 
+(* generate the projection of the i-th field of a n-tuple *)
+let project_tuple ~dbg tuple n i =
+  let module P = Flambda_primitive in
```

Comment:
This alias is already defined in `Simplify_import`

Reply:
Alias removed